class.photon.php

```diff
@@ -473,6 +473,34 @@ public function filter_image_downsize( $image, $attachment_id, $size ) {
 			return $image;
 		}
 
+		// Similar to the need to skip the admin, we should skip the REST API to avoid Photonizing images out of context.
+		if ( defined( 'REST_REQUEST' ) && REST_REQUEST ) {
+			global $wp;
+			if ( false !== strpos( $wp->query_vars['rest_route'], '/wp/v2/media/' ) && ! empty( $_REQUEST['context'] ) && 'edit' === $_REQUEST['context'] && // phpcs:ignore WordPress.Security.NonceVerification.NoNonceVerification
+				/**
+				 * Provide plugins a way of running Photon for images for the REST API (media endpoint).
+				 *
+				 * Note: enabling this will result in Photon URLs added to REST API responses in the edit context, which breaks Gutenberg, and possibly other REST API consumers.
+				 *
+				 * @module photon
+				 *
+				 * @since 6.7.1
+				 *
+				 * @param bool false Allow Photon to run on the REST API media endpoint. Default to false.
```

Comment:
This filter claims to be able to adjust photon settings during REST API requests, but it only fires if `?context=edit`.